netdata.spec.in

```diff
@@ -69,12 +69,14 @@ happened, on your systems and applications.
 
 %install
 rm -rf $RPM_BUILD_ROOT
-%{__make} %{?_smp_mflags} install \
-	DESTDIR=$RPM_BUILD_ROOT
+%{__make} %{?_smp_mflags} DESTDIR=$RPM_BUILD_ROOT install
+
+find $RPM_BUILD_ROOT -name .keep -print0 | xargs --null --no-run-if-empty rm
```

Comment:
This find command greatly increases the flexibility, for example:

- if paths contain spaces, this will now work as before it would not have.
- expanded `xargs` options so it's easier to read/understand by someone who has no idea what the hell `xargs` is.